ctutil/ctutil_test.go

```diff
@@ -54,38 +54,38 @@ func TestLeafHash(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		// Parse chain
-		chain, err := x509util.CertificatesFromPEM([]byte(test.chainPEM))
-		if err != nil {
-			t.Errorf("%s: error parsing certificate chain: %s", test.desc, err)
-			continue
-		}
+		t.Run(test.desc, func(t *testing.T) {
+			// Parse chain
+			chain, err := x509util.CertificatesFromPEM([]byte(test.chainPEM))
+			if err != nil {
+				t.Fatalf("%s: error parsing certificate chain: %s", test.desc, err)
```

Comment:
`test.desc` is still prefixed on all of these error messages when it doesn't need to be.

Reply:
Done.

Reply:
It doesn't appear to be done...

Reply:
Sorry, only got a subset of them. Should be done now.